src/zappy_ai_src/ai_handle_response.py

```diff
@@ -99,10 +99,14 @@ def __countItem(self, tileStr : str, searchedItem : str) -> int:
         """ Count the occurence of the Item in the Tile """
         return tileStr.count(searchedItem)
 
-    def GetTile(self, x : int, y : int = 0) -> Tile:
+    def GetTile(self, x : int, y : int) -> Tile:
         """" Get one Tile of the Map """
         return self.__map[x + (y * y)]
 
+    def GetTile(self, x) -> Tile:
+        """" Get one Tile of the Map """
+        return self.__map[x]
+
     def GetMapSize(self) -> int:
         """" Get the size of the Map """
         return len(self.__map)
```

Comment:
Python  is easy to use, i don't understand how

```suggestion
     def GetMapSize(self) -> int:
         """" Get the size of the Map """
         return len(self.__map), len(self.__map[0])
```
Is hard to write